manifests/params.pp

```diff
@@ -143,6 +143,7 @@
       } else {
         $detach_service_in_init = true
       }
+      $manage_memory_cgroup = true
```

Comment:
This will fail with strict variables on as the variable in question is referenced in the init but not set for other operating systems, You'll need to include it set to false for everything else I'm afraid.